keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1870,21 +1849,17 @@ type DeviceRequestAllocationResult struct {
     // +required
     Device string
 
-    // AdminAccess is a copy of the AdminAccess value in the
-    // request which caused this device to be allocated.
-    //
-    // New allocations are required to have this set when the DRAAdminAccess
-    // feature gate is enabled. Old allocations made
-    // by Kubernetes 1.31 do not have it yet. Clients which want to
-    // support Kubernetes 1.31 need to look up the request and retrieve
-    // the value from there if this field is not set.
+    // AdminAccess indicates that this device was allocated for
+    // administrative access. See the corresponding request field
+    // for a definition of mode.
     //
     // This is an alpha field and requires enabling the DRAAdminAccess
-    // feature gate.
+    // feature gate. Admin access is disabled if this field is unset or
+    // set to false, otherwise it is enabled.
     //
-    // +required
+    // +optional
```

Comment:
To reflect the latest https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/resource/v1beta1/types.go

Reply:
should we update any stale API definitions in a separate PR so there isn't a suggesting that KEP-5018 is responsible for the change from required to optional?

Reply:
A separate commit for this change is good enough for me.

Reply:
Once this https://github.com/kubernetes/enhancements/pull/5087 is merged, this diff will go away.